### PR TITLE
Change amount color in BRBHistoryViewController

### PR DESCRIPTION
**Pull request by ReadeP**
## Summary
Originally there was no way to differentiate between positive transactions (money enters user's account) and negative transactions (money leaves user's account). These changes show incoming money as green and positive.

## Changes Made
- Updated brbQuery.graphql by adding positive attribute to history
- Added 'positive' boolean attribute to BRBAccount model
- Updated brbHistory array in NetworkManager to return BRBHistory object with a value for the positive boolean
- Updated the configuration of BRBHistoryTableViewCell to check the positive value and if positive, display the amount in green and if negative, display it in red

## Test Coverage
I tested these changes initially by hardcoding the positive values for each cell based on a random boolean against my own account since I did not received any money in my account. I also tested it against William's account which had positive transactions.

## Screenshots
Simulator Screen Shot - iPhone 11 - 2019-10-16 at 18 12 10
Simulator Screen Shot - iPhone 11 - 2019-10-16 at 18 12 06
Simulator Screen Shot - iPhone 11 - 2019-10-16 at 18 12 08

### CODE REVIEW

File: Eatery/Views/BRB/BRBHistoryTableViewCell.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ class BRBHistoryTableViewCell: UITableViewCell {
     private var titleLabel: UILabel!
     private var subtitleLabel: UILabel!
     private var priceLabel: UILabel!
+    private var positive: Bool!
```
Comment (whoiswillma) [inline Eatery/Views/BRB/BRBHistoryTableViewCell.swift]: We don't need this property; the label text color can be set inside the `configure` method.

File: Eatery/Views/BRB/BRBHistoryTableViewCell.swift [hunk 2]
```diff
@@ -53,10 +53,18 @@ class BRBHistoryTableViewCell: UITableViewCell {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func configure(title: String, subtitle: String, amount: String) {
-        titleLabel.text = title
-        subtitleLabel.text = subtitle
-        priceLabel.text = "-$\(amount)"
+    func configure(title: String, subtitle: String, amount: String, positive: Bool) {
+        self.titleLabel.text = title
```
Comment (whoiswillma) [inline Eatery/Views/BRB/BRBHistoryTableViewCell.swift]: Can we get rid of the `self.`?

**Review by whoiswillma — APPROVED** [review]
Looks good!

File: Eatery/Controllers/BRB/BRBAccountViewController.swift [hunk 3]
```diff
@@ -86,7 +86,7 @@ extension BRBAccountViewController: UITableViewDataSource {
         let historyItem = account.history[indexPath.row]
         cell.configure(title: historyItem.name,
                        subtitle: historyItem.timestamp,
-                       amount: historyItem.amount)
+                       amount: historyItem.amount, positive: historyItem.positive)
```
Comment (kevinchan159) [inline Eatery/Controllers/BRB/BRBAccountViewController.swift]: can we move `positive: historyItem.positive` onto its own line like the other arguments? i.e. it should look like
```
cell.configure(
  title: historyItem.name,
  subtitle: historyItem.timestamp,
  amount: historyItem.amount
  positive: historyItem.positive
)
```

**Review by kevinchan159 — APPROVED** [review]
amazing!!! 🥳

**Pull request merged**